apps/timetable/admin/index.html

```diff
@@ -1,3 +1,4 @@
+<!--[if IE 9]> <html class="lt-ie10"> <![endif]-->
 <html>
```

Comment:
Just wondering, shouldn't we add `<!--[if !IE 9]> -->` around this `<html>` tag then to not end up with 2 html tags?